tests/web.rs

```diff
@@ -10,32 +10,102 @@ wasm_bindgen_test_configure!(run_in_browser);
 
 #[wasm_bindgen_test]
 fn account_address_from_string() {
-    let address = Address::from_string("ca1q09u0nxmnfg7af8ycuygx57p5xgzmnmgtaeer9xun7hly6mlgt3pjyknplu");
+    let address =
+        Address::from_string("ca1q09u0nxmnfg7af8ycuygx57p5xgzmnmgtaeer9xun7hly6mlgt3pjyknplu");
```

Comment:
`from_string` is going to be `from_string_anyprefix` at the next update of chain-libs